src/lib/Scenes/Collection/Collection.tsx

```diff
@@ -96,7 +96,7 @@ export const CollectionContainer = createFragmentContainer(Collection, {
       slug
       ...CollectionHeader_collection @arguments(screenWidth: $screenWidth)
       ...CollectionArtworks_collection
-      ...FeaturedArtists_collection @arguments(screenWidth: $screenWidth)
```

Comment:
The `ArtistListItem` component uses the original-sized image. A follow-up PR could change that to use the resized version, like we were originally doing in `FeaturedArtists`.